_stbt/core.py

```diff
@@ -687,8 +687,11 @@ def raise_in_user_thread(exception):
         display[0].tell_user_thread(exception)
     mainloop = _mainloop()
 
-    sink_pipeline = SinkPipeline(
-        gst_sink_pipeline, raise_in_user_thread, save_video)
+    if not gst_sink_pipeline and not save_video:
+        sink_pipeline = NoSinkPipeline()
+    else:
+        sink_pipeline = SinkPipeline(  # pylint: disable=redefined-variable-type
+            gst_sink_pipeline, raise_in_user_thread, save_video)
```

Comment:
What happens if `gst_sink_pipeline` is `None` or `""` or `"disable"` but `save_video` is truthy?

Should we fall back to fakesink + a pipeline that encodes the video? Should we error?

Maybe instead of "disable" we can grep the sink_pipeline for "fakesink" -- something like `if "fakesink" in gst_sink_pipeline and not save_video`. Maybe that's too magical though. It would break if someone put in a tee with a fakesink on one branch and something useful on the other, but it seems unlikely that anyone would do that.